components/cereal-service/habitat/hooks/run

```diff
@@ -0,0 +1,10 @@
+#!{{pkgPathFor "core/bash"}}/bin/bash -e
+
+exec 2>&1
+
+# Call the script to block until user accepts the MLSA via the package's config
+{{pkgPathFor "chef/mlsa"}}/bin/accept {{cfg.mlsa.accept}}
+
+pg-helper ensure-service-database "cereal_service"
```

Comment:
Did we stop prefacing db's with `chef_`? Or am I imagining that we do that?

Reply:
i think you're right